scripts/core/ui/components/alert.scss

```diff
@@ -0,0 +1,17 @@
+.alert-extended {
+    display: flex!important;
+    width: 100%!important;
+    justify-content: space-between!important;
+    align-items: center!important;
+    padding-left: 1.2rem!important;
```

Comment:
These could be replaced with the `spacing` classes from the UI-Framework:
https://ui-framework.superdesk.org/#/components/spacing

In this case you would add the classes:
`sd-padding-x--1-5` and  `sd-margin-b--0`

This has the benefit that if the UI-Framework ever changes the `$sd-base-increment` variable, views like this would continue to be consistent with the rest of the UI-Framework